main.go

```diff
@@ -507,10 +520,12 @@ Do you really want to delete?
 `, strings.Join(names, "\n"))
 					interactive := c.GlobalBool("interactive")
 					if !interactive || interactive && askForConfirmation(msg) {
-						return state.DeleteReleases(helm, true)
+						return state.DeleteReleases(&affectedReleases, helm, true)
 					}
 					return nil
 				})
+				affectedReleases.DisplayAffectedReleases(c.App.Metadata["logger"].(*zap.SugaredLogger))
```

Comment:
Just a note: I think I'll be changing all the verbose log messages from helmfile to be sent via the zap logger connected to STDERR,
and useful messages like ones printed by `DisplayAffectedReleases` to be redirected to STDOUT.